src/permission-set.js

```diff
@@ -736,7 +738,10 @@ class PermissionSet {
     if (!webClient) {
       return Promise.reject(new Error('Cannot save - no web client'))
     }
-    return webClient.put(aclUrl, this.serialize(rdf))
+    contentType = contentType || DEFAULT_CONTENT_TYPE
+    let graph = this.serialize(rdf, contentType)
+    return Promise.resolve()
```

Comment:
Why are you returning `Promise.resolve()`? Shouldn't this just be `return webClient.put(aclUrl, graph, contentType)`